src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -48,6 +51,8 @@ interface BuildLocator {
      */
     fun withAllBranches() : BuildLocator
 
+    fun withSinceBuild(locator: BuildLocator): BuildLocator
```

Comment:
Is it really important to have one request instead of two? Could you please describe your use case?
If it's needed we can provide API for that, though I would take extension function for BuildLocator as an argument to avoid casting to implementation.